storm-core/src/jvm/backtype/storm/utils/Utils.java

```diff
@@ -135,35 +137,68 @@ public static void sleep(long millis) {
     }
 
     public static Map findAndReadConfigFile(String name, boolean mustExist) {
+        InputStream in = null;
+        Boolean confFileEmpty = false;
```

Comment:
@lazyval  I know about primitives and objects in java . Yes makes sense to use a boolean.